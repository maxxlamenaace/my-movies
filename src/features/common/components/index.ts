import Loader from './loader/Loader';
import Logo from './logo/Logo';
import Footer from './footer/Footer';
import AppBar from './app-bar/AppBar';
import Container from './container/Container';
import ScrollableAppBar from './scrollable-app-bar/ScrollableAppBar';
import Sidebar from './sidebar/Sidebar';
import ManualSwiper from './manual-swiper/ManualSwiper';
import Wrapper from './wrapper/Wrapper';
import PaginatorSwiper from './paginator-swiper/PaginatorSwiper';

export {
  Loader,
  Logo,
  Footer,
  AppBar,
  Container,
  ScrollableAppBar,
  Sidebar,
  ManualSwiper,
  Wrapper,
  PaginatorSwiper,
};
