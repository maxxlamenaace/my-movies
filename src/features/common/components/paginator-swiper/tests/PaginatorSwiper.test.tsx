import React from 'react';
import { render } from '@testing-library/react';

import { PaginatorSwiper } from '@/features/common';

describe('PaginatorSwiper', () => {
  it('renders correctly', () => {
    const { asFragment, container } = render(
      <PaginatorSwiper>
        <div id='children' />
      </PaginatorSwiper>,
    );

    expect(container.querySelector('#children')).toBeDefined();
    expect(asFragment()).toMatchSnapshot();
  });
});
