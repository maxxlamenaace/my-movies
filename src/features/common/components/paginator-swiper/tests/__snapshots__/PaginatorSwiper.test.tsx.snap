// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginatorSwiper renders correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-10mh0q4"
  >
    <div
      class="swiper swiper-initialized swiper-horizontal swiper-pointer-events"
      id="paginator-swiper"
      style="width: 100%;"
    >
      <div
        class="swiper-wrapper"
        style="cursor: grab; transition-duration: 0ms; transform: translate3d(NaNpx, 0px, 0px);"
      />
      <div
        class="swiper-button-prev"
      />
      <div
        class="swiper-button-next"
      />
      <div
        class="swiper-pagination swiper-pagination-clickable swiper-pagination-bullets swiper-pagination-horizontal"
      />
      <div
        id="children"
      />
    </div>
  </div>
</DocumentFragment>
`;
